File: app/src/main/java/com/pavelprymak/propodcast/presentation/adapters/FavoriteEpisodeClickListener.java
package com.pavelprymak.propodcast.presentation.adapters;

import android.view.View;

import com.pavelprymak.propodcast.model.db.FavoriteEpisodeEntity;

public interface FavoriteEpisodeClickListener {
    void onEpisodeItemClick(FavoriteEpisodeEntity episodesItem);

    void onEpisodeMoreOptionsClick(String episodeId, String link, View v);
}

File: app/src/main/java/com/pavelprymak/propodcast/presentation/adapters/FavoriteEpisodeClickListener.kt
package com.pavelprymak.propodcast.presentation.adapters

import android.view.View

import com.pavelprymak.propodcast.model.db.FavoriteEpisodeEntity

interface FavoriteEpisodeClickListener {
    fun onEpisodeItemClick(episodesItem: FavoriteEpisodeEntity)

    fun onEpisodeMoreOptionsClick(episodeId: String, link: String?, v: View)
}
